load.php

```diff
@@ -236,6 +236,15 @@ function perflab_can_load_module( $module ) {
 		return true;
 	}
 
+	$standalone_plugins_constants = perflab_get_standalone_plugins_constants();
+	if (
+		isset( $standalone_plugins_constants[ $module ] ) &&
+		defined( $standalone_plugins_constants[ $module ] ) &&
+		0 !== strpos( constant( $standalone_plugins_constants[ $module ] ), 'Performance Lab ' )
```

Comment:
Nit-pick: You can simplify this by using `str_starts_with()`, which is polyfilled regardless of PHP version since WordPress 5.9.
```suggestion
		! str_starts_with( constant( $standalone_plugins_constants[ $module ] ), 'Performance Lab ' )
```

Reply:
Thanks, Felix. I made the same changes initially, but when I ran `npm run lint-php` it gave me the error `Fatal error: Uncaught TypeError: vsprintf(): Argument #2 ($values) must be of type array, string given in /app/vendor/squizlabs/php_codesniffer/src/Files` so i updated it with `strpos` but now it working.